dp3t-sdk/sdk/src/main/java/org/dpppt/android/sdk/internal/backend/ReportService.java

```diff
@@ -21,11 +21,18 @@
 interface ReportService {
 
 	@Headers("Accept: application/json")
-	@POST("v1/gaen/exposed")
-	Call<Void> addGaenExposee(@Body GaenRequest exposeeRequest, @Header("Authorization") String authorizationHeader);
+	@POST("v2/gaen/exposed")
```

Comment:
If we switch to v2 here, we have to completely remove exposednextday, cause we will never receive a token for a next day upload on the v2 endpoint.